recipes/atari-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "atari-py" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6249ad5079b0489e87eb44e65485bb1b07cc1b5af729f1ee52ece749503ceb1d
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - pip
+    - python
+    - zlib
+  run:
+    - python
+    - numpy
+    - six
+    - zlib
+
+test:
+  imports:
+    - atari_py
+  requires:
+    - nose2
+
+about:
+  home: https://github.com/openai/atari-py
+  license: GPL-2
```

Comment:
I think you will need to add a spdx complaint license designator:

https://spdx.org/licenses/

Reply:
@synapticarbors this is the license designator https://spdx.org/licenses/GPL-2.0-or-later.html and it is used in various recipes in the conda forge as seen here: https://github.com/search?q=GPL++user%3Aconda-forge&type=Code

Reply:
the `-or-later` isn't usually required, although i added it here to be safe :)